xml_converter/src/attribute/trail_data.cpp

```diff
@@ -21,41 +21,52 @@ using namespace std;
 TrailData parse_trail_data(rapidxml::xml_attribute<>* input, vector<XMLError*>* errors, string base_dir) {
     TrailData trail_data;
     trail_data.trail_data_relative_path = get_attribute_value(input);
-    if (base_dir == "" || trail_data.trail_data_relative_path == "") {
-        errors->push_back(new XMLAttributeValueError("", input));
+    if (base_dir == "") {
+        errors->push_back(new XMLAttributeValueError("Marker pack file name is empty", input));
```

Comment:
Is this actually an `XMLAttributeValueError` on the `input` token?

It seems like this is an error entirely unrelated to the XML even.